### PR TITLE
BigQuery: Adds sample for creating a table with a schema

### PR DESCRIPTION
**Pull request by alixhami**
Resolves https://github.com/GoogleCloudPlatform/ruby-docs-samples/issues/246

### CODE REVIEW

**Comment by frankyn** [comment]
Passing PR review for BD/ML to @remi.

File: bigquery/tables.rb [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-def create_table project_id:, dataset_id:, table_id:
+def create_table_with_schema project_id:, dataset_id:, table_id:
   # [START create_table]
```
Comment (tswast) [inline bigquery/tables.rb]: Per our conversation yesterday, you're going to update the region tags in a separate PR, right?

File: bigquery/tables.rb [hunk 2]
```diff
@@ -23,12 +23,32 @@ def create_table project_id:, dataset_id:, table_id:
   bigquery = Google::Cloud::Bigquery.new project: project_id
   dataset  = bigquery.dataset dataset_id
 
-  dataset.create_table table_id
+  table    = dataset.create_table table_id do |schema|
```
Comment (tswast) [inline bigquery/tables.rb]: Maybe change `|schema|` to `|updater|` or `|table|`?

The block for `create_table` is actually a "Table::Updater" (I see that's not documented in the API docs, so I just created https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/2038). It lets you set any property on the table, so more than just add schema fields. \Ffor example, it'll be how you set the encryption key once that feature is released).

File: bigquery/tables.rb [hunk 3]
```diff
@@ -23,12 +23,32 @@ def create_table project_id:, dataset_id:, table_id:
...
+    schema.string  "full_name", mode: :required
+    schema.integer "age",       mode: :required
+  end
 
   puts "Created table: #{table_id}"
   # [END create_table]
 end
 
+def create_table_without_schema project_id:, dataset_id:, table_id:
+  # [START create_table_without_schema]
```
Comment (tswast) [inline bigquery/tables.rb]: Missing `bigquery_` in this region tag (I see it in END but not at START)

File: bigquery/tables.rb (hunk 1 above)
Comment (alixhami) [reply bigquery/tables.rb]: Yeah it currently doesn't have a branch specified, so changing the region tag now would break the link

**Pull request merged**